theano/gof/tests/test_reallocation.py

```diff
@@ -0,0 +1,47 @@
+"Test of reduce allocation"
```

Comment:
We should not create a new file when not needed. The tests in this file should go in:

compile/tests/test_vm.py as it test the code in compile/vm.py